### PR TITLE
Update enable-exploit-protection.md

### PR DESCRIPTION
**Pull request by kurtsarens**
Audit of mitigations is not always available via PS but is with other management options

### CODE REVIEW

File: windows/security/threat-protection/microsoft-defender-atp/enable-exploit-protection.md [hunk 1]
```diff
@@ -225,20 +225,20 @@ Code integrity guard | App-level only |   BlockNonMicrosoftSigned,   AllowStoreS
...
-Validate stack integrity (StackPivot) | App-level only |   EnableRopStackPivot  | Audit not available
+Validate stack integrity (StackPivot) | App-level only |   EnableRopStackPivot  | Audit not available<a href="#r2" id="t2">\[2\]</a>
 
 <a href="#t1" id="r1">\[1\]</a>: Use the following format to enable EAF modules for DLLs for a process:
 
 ```PowerShell
 Set-ProcessMitigation -Name processName.exe -Enable EnableExportAddressFilterPlus -EAFModules dllName1.dll,dllName2.dll
 ```
-
+<a href="#t2" id="r2">\[2\]</a>: Audit for this mitigation is not available via Powershell CmdLet.
```
Comment (illfated) [inline windows/security/threat-protection/microsoft-defender-atp/enable-exploit-protection.md:241]: Codestyle shows that it is more common to use lowercase 'cmdlet' than to use CamelCase:
```suggestion
<a href="#t2" id="r2">\[2\]</a>: Audit for this mitigation is not available via Powershell cmdlets.
```

**Comment by kurtsarens** [comment]
can this be published?

**Review by illfated — APPROVED** [review]
Looks good to me.

**Comment by illfated** [comment]
#sign-off

**Comment by illfated** [comment]
@e0i , for your information.

**Review by denisebmsft — APPROVED** [review]
Thank you @kurtsarens

**Pull request merged**